stanozzy/stackoverflow
# create a sign up page

ft-sign-up-162557854
*What does this PR do*
Builds the sign up page for new users
*Description of task to be completed*
• Write HTML for the user sign up page
• Write CSS to style the sign up page
*How should this be manually tested*

1. Clone this repository
2. Cd into the StackOverflow-lite folder
3. Open register.html from browser

*What are the relevant pivotal tracker stories*
#162557854 A user should be able to create a StackOveflow-lite account

![screenshot from 2018-12-15 23-26-16](https://user-images.githubusercontent.com/45230437/50047239-7e4d9f00-00a9-11e9-9f7d-928714a6274c.png)
